test/e2e/storage/testsuites/subpath.go

```diff
@@ -901,5 +905,9 @@ func formatVolume(f *framework.Framework, pod *v1.Pod) {
 }
 
 func podContainerExec(pod *v1.Pod, containerIndex int, bashExec string) (string, error) {
```

Comment:
nit: variable should be `scriptCmd` or something instead of `bashExec` now

Reply:
fixed